spec/money/formatting_rules_spec.rb

```diff
@@ -14,4 +14,22 @@
     expect(rules).to eq(separator: '.')
     expect(rules).not_to eq(new_rules)
   end
+
+  context 'when the position is :before' do
+    it 'warns about deprecated :symbol_position' do
+      expect_any_instance_of(Money::FormattingRules).to receive(:warn)
```

Comment:
Note: We're aware that using `any_instance_of` may not be great - but given that the deprecation code is called on initialize and the spec helper warning modules are noops, it is difficult to assert against the deprecation warning strings.  

We are up for alternatives on testing!